packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -79,86 +83,91 @@ class FrameFlameChart extends CoreElement {
   }
 
   void _render(TimelineFrame frame) {
-    const int leftIndent = 80;
+    const int leftIndent = 70;
     const int rowHeight = 25;
 
-    // TODO(kenzie): re-write this scale logic.
-    const double microsPerFrame = 1000 * 1000 / 60.0;
-    const double pxPerMicro = microsPerFrame / 1000.0;
+    // 16,666 microseconds / frame will achieve a frame rate of 60 FPS.
+    const double targetMicrosPerFrame = 1000 * 1000 / 60.0;
 
-    int row = 0;
+    // Pixels per microsecond in order to fit a single frame in 1500px. In order
+    // for the whole frame to fit in 1500px at this drawing ratio, the frame
+    // duration must be [targetMicrosPerFrame] or less. 1500px is arbitrary.
+    const double pixelsPerMicro = 1500 / targetMicrosPerFrame;
 
-    final int microsAdjust = frame.startTime;
+    final int frameStartOffset = frame.startTime;
 
-    int maxRow = 0;
+    // Add 15 to account for vertical spacing between the CPU and GPU sections.
+    final cpuSectionHeight = frame.cpuEventFlow.getDepth() * rowHeight + 15;
```

Comment:
make 15 a constant with a name and then you don't need a comment.

Reply:
done